### PR TITLE
Added subfooter

### PR DESCRIPTION
**Pull request by kdzapp**
Fixes # .
#40 

### PR Description
The thing above the footer

### What needs to be reviewed?
Everything, I coded it but it's being buggy and sometimes the whole page doesn't show up

### Context?
n/a

### CODE REVIEW

File: app/pages/blackout/index.jsx [hunk 1]
```diff
@@ -1,42 +1 @@
...
                <Sponsorship />
                <QuoteSection />
                <SubFooter />
                <Footer />
            </Page>
        );
    }
}

export default BlackoutPage;
```
Comment (roball24) [inline app/pages/blackout/index.jsx]: lol what happened here "^M"

File: app/pages/blackout/subfooter.jsx [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+        return (
+            <Container>
+                <Flexer>
+                    <Text> Turn your ideas into reality. </Text>
+                    <TextSubmitPositioner>
+                            <StyledInput
+                                color={this.props.theme.darkPink}
+                                borderColor="white"
+                                placeholder="ENTER YOUR EMAIL FOR UPDATES"
+                                placeholderColor={this.props.theme.darkPink}
```
Comment (roball24) [inline app/pages/blackout/subfooter.jsx]: you need to mapStateToProps to use the redux theme

File: app/pages/blackout/subfooter.jsx [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+                                feedbackColor="white"
+                                feedback={this.props.status.message}
```
Comment (roball24) [inline app/pages/blackout/subfooter.jsx]: same as above

File: app/pages/blackout/subfooter.jsx [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+        this.state = {
+            email: '',
+            feedback: ''
+        };
+        this.emailChange = this.emailChange.bind(this);
+        this.submitEmail = this.submitEmail.bind(this);
+    }
+
+    componentDidMount() {
+        this.props.dispatch(push(routes.SUBSCRIBE));
```
Comment (roball24) [inline app/pages/blackout/subfooter.jsx]: this component shouldn't manage routes

File: app/pages/blackout/subfooter.jsx [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+    }
+
+    emailChange(e) {
+        this.setState({
+            email: e.target.value
+        });
+    }
+
+    submitEmail() {
+        this.props.dispatch(SubscribeThanks.subscribe(this.state.email));
```
Comment (roball24) [inline app/pages/blackout/subfooter.jsx]: 'SubscribeThunks'

File: app/pages/blackout/subfooter.jsx [hunk 6]
```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+import styled from 'styled-components';
+
+import { InputText, Container } from '../../components';
+import { devices } from '../../styles';
+import { push } from 'react-router-redux';
+import { routes } from '../../constants';
+import { SubscribeThanks } from '../../actions';
```
Comment (roball24) [inline app/pages/blackout/subfooter.jsx]: 'SubscribeThunks'

**Comment by antoniomika** [comment]
I'm gonna close this because ¯\_(ツ)_/¯

**Pull request closed without merging**